frontend/src/metabase/collections/components/PinnedItemOverview/PinnedItemOverview.styled.tsx

```diff
@@ -23,16 +22,8 @@ export const Grid = styled.div`
   }
 `;
 
-export interface SectionHeaderProps {
-  hasTopMargin?: boolean;
-}
-
-export const SectionHeader = styled.div<SectionHeaderProps>`
-  margin-top: ${props => (props.hasTopMargin ? "1.5rem" : "")};
-  padding-bottom: 1.15rem;
-`;
-
-export const SectionSubHeader = styled.div`
-  color: ${color("text-medium")};
-  padding-top: 0.5rem;
+export const SectionContainer = styled.div`
+  &:not(:first-of-type) {
```

Comment:
Can this live in the [parent's `gap` property`](https://github.com/metabase/metabase/blob/cbc56ffcf9f3cc244e77c26838e276266e16e12f/frontend/src/metabase/collections/components/PinnedItemOverview/PinnedItemOverview.styled.tsx#L11), ie: 
```css
gap: 1.5rem ${GAP_REM}rem
```
on the `Container` would allow you to not have this `first-of-type` here.